homeassistant/components/dominos.py

```diff
@@ -0,0 +1,111 @@
+import json
+import logging
+import voluptuous as vol
+import homeassistant.helpers.config_validation as cv
+from pizzapi import Customer, Address, Order
+import unicodedata
+
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# The domain of your component. Should be equal to the name of your component.
+DOMAIN = 'dominos'
+
+ATTR_COUNTRY = 'country_code'
+ATTR_FIRST_NAME = 'first_name'
+ATTR_LAST_NAME = 'last_name'
+ATTR_EMAIL = 'email'
+ATTR_PHONE = 'phone'
+ATTR_ADDRESS = 'address'
+ATTR_ORDERS = 'orders'
+ATTR_DUMP_MENU = 'dump_menu'
+ATTR_ORDER_ENTITY = 'order_entity_id'
+
+REV = '183d0f5522e7fa99776e8c6c692b56e69f0e6b8f'
+REQUIREMENTS = [
+    'https://github.com/wardcraigj/pizzapi/archive/%s.zip#pizzapi==0.0.2'
```

Comment:
Please publish this to PyPi

Reply:
Does this work in every country?

Reply:
Right now, this works in the US and Canada. Currently, the original author of the client library has not put it on pypi. I noticed there were a few other components that handled their dependencies like this, so I thought it would be ok to handle it like this, but I can get in touch with the author if it isn't